test/scenarios/el94-base@upgrade-ok.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I don't see any removals in this dir, shouldn't we remove scenarios on el92 and el93 unless they upgrade to el94?